src/cmd_line/commands/file.ts

```diff
@@ -85,6 +86,19 @@ export class FileCommand extends node.CommandBase {
       let folder = vscode.Uri.file(newFilePath);
       await vscode.commands.executeCommand("vscode.open", folder,
         this._arguments.position === FilePosition.NewWindow ? this.getViewColumnToRight() : this.getActiveViewColumn());
+
+      await vscode.commands.executeCommand("cursorMove", {
```

Comment:
It's no problem to do `:50` in the same file, as we can just move the cursor ourselves. The problem is that once we call the API to open a new file, we don't have access to the new modeHandler yet, in this file.

Come to think of it though, there was another part of the code where we might have needed to do something similar. I'll take a look at that later.